src/installation-info/ck/getCKPremiumBundleInstallationInfo.ts

```diff
@@ -0,0 +1,27 @@
+/**
+ * @license Copyright (c) 2003-2024, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+import type { BundleInstallationInfo } from '../types';
+
+import { isSemanticVersion } from '@/utils/isSemanticVersion';
+import { isCKCdnPremiumBundlePackInjected } from '@/cdn/ck/isCKCdnPremiumBundlePackInjected';
+
+/**
+ * Returns information about the premium CKEditor bundle installation.
+ */
+export function getCKPremiumBundleInstallationInfo(): BundleInstallationInfo | null {
+	const { CKEDITOR_VERSION, CKEDITOR_PREMIUM_FEATURES } = window;
+
+	if ( !CKEDITOR_PREMIUM_FEATURES || !isSemanticVersion( CKEDITOR_VERSION ) ) {
```

Comment:
The npm installation doesn't register the `CKEDITOR_PREMIUM_FEATURES` global, so it will always return `null`?

Reply:
good catch!